.github/ISSUE_TEMPLATE/issue-template.md

```diff
@@ -4,16 +4,26 @@ about: How to create an issue for this repository
 
 ---
 
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+
 If you have any questions, feel free to create an issue with the tag [question].  
 If you wish to suggest an enhancement or feature request, add the tag [feature request].  
-If you are submitting a bug report, please fill in the following details.  
+If you are submitting a bug report, please fill in the following details.
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
 **Code example**
 Please try to provide a minimal example to reproduce the bug. Error messages and stack traces are also helpful.
 
+Please use the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks)
+for both code and stack traces.
+
+```python
+from stable_baselines import ...
+
+```
+
```

Comment:
Well, I cant add this to suggestions (due to the ```),

I would add
`` ```bash
Traceback (most recent call last):
  File ...
``` ``

so that the users remeber to format the stacktrace as well